pkg/parsers/parsers.go

```diff
@@ -196,3 +196,53 @@ func ParseLink(val string) (string, string, error) {
 	}
 	return arr[0], arr[1], nil
 }
+
+// ParseIntList parses and validates the specified string as the value
+// found in some cgroup file (e.g. `cpuset.cpus`, `cpuset.mems`), which could be
+// one of the formats below. Note that duplicates are actually allowed in the
+// input string. It returns a `map[int]bool` with available elements from `val`
+// set to `true`.
+// Supported formats:
+//     7
+//     1-6
+//     0,3-4,7,8-10
+//     0-0,0,1-7
+//     03,1-3      <- this is gonna get parsed as [1,2,3]
+//     3,2,1
+//     0-2,3,1
+func ParseIntList(val string) (map[int]bool, error) {
```

Comment:
Perhaps make this ParseUintList, since negative numbers won't give the same result?